<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string> </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Use?</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string> </string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Sample</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flat Before</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flat After</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark Before</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dark After</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>180 degree</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sample / Spectra Log</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flat Before Log</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flat After Log</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="step_preview">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="step_all">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="expectedResourcesLabel">
     <property name="text">
      <string>Placeholder txt for Developer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QComboBox" name="pixel_bit_depth">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>float32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float64</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="images_are_sinograms">
       <property name="text">
        <string>Images are sinograms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_pixel_bit_depth">
       <property name="text">
        <string>Pixel Bit Depth:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="pixelSize">
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Set pixel size to automatically compute linear attenuation after reconstruction. If left 0 the reconstruction will be output in gray values</string>
       </property>
       <property name="text">
        <string>Pixel size (microns)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
